apps/api/src/services/agent-manager.service.ts

```diff
@@ -983,7 +983,10 @@ export class AgentManager {
    * @returns The sanitized agent object
    */
   sanitizeAgent(agent: SelectAgent) {
-    return AgentPublicSchema.parse(agent);
+    return AgentPublicSchema.parse({
+      ...agent,
+      isVerified: !!agent.walletAddress,
```

Comment:
This is the only business logic change. In general as long as our controllers use the `sanitizeAgent` method, the responses will match the spec schema